tests/nest_tests/test_nest_math_function.py

```diff
@@ -56,7 +56,10 @@ def test_math_function(self):
         log10_state_specifier = "log10_state"
         erf_state_specifier = "erf_state"
         erfc_state_specifier = "erfc_state"
-        nest.SetStatus(mm, {"record_from": ["x", ln_state_specifier, log10_state_specifier, erf_state_specifier, erfc_state_specifier]})
+        ceil_state_specifier = "ceil_state"
+        floor_state_specifier = "floor_state"
+        round_state_specifier = "round_state"
+        nest.SetStatus(mm, {"record_from": ["x", ln_state_specifier, log10_state_specifier, erf_state_specifier, erfc_state_specifier, ceil_state_specifier, floor_state_specifier, round_state_specifier]})
```

Comment:
I don't quite understand this line. Maybe others feel the same and a comment would help? Thanks!

Reply:
I think the names of the recordables are assigned to variables and they are specified in the `SetStatus()` instead of directly providing their names as literal strings to the function. I also think that the former is not necessary.

Reply:
Ah! Thanks for the clarification. Maybe it would be better to remove the variable assignments and use the specifiers directly?